### DISCUSSION TITLE
raspberry pi 1 support

### DISCUSSION THREAD

**Discussion by Rubenpii** [bpatrik/pigallery2#864] [Q&A] [answered]
I am trying to use pigallery2 on a raspberry pi 1. I am following the README for setup with docker: https://github.com/bpatrik/pigallery2/blob/master/docker/README.md

However, there is no support for linux/arm/v6. When trying to pull the docker image it tells me there is no tag that supports v6:
`no matching manifest for linux/arm/v6 in the manifest list entries`. Is this something that might be supported at some point or should I try the direct install with node and npm? Is that confirmed to work on a raspberry pi 1?

**Comment by bpatrik** [comment, marked as answer]
You can try to build the docker file yourself on rpi1.

That is less documented/tested but should not be too hard .

There is also a self contained docker file that should be easier to build
(but I haven't tested for a while)


-- Sorry for being brief, sent from my phone.

**Reply by Rubenpii** [reply]
Building the Alpine docker file on the rpi1 itself worked out fine. The build took quite a while, but that's because of the limited hardware of rp1, pigallery2 itself runs smoothly!
